internal/deviations/deviations.go

```diff
@@ -285,13 +310,23 @@ func MissingValueForDefaults(_ *ondatra.DUTDevice) bool {
 }
 
 // TraceRouteL4ProtocolUDP returns if device only support UDP as l4 protocol for traceroute.
-func TraceRouteL4ProtocolUDP(_ *ondatra.DUTDevice) bool {
-	return *traceRouteL4ProtocolUDP
+// Default value is false.
+func TraceRouteL4ProtocolUDP(dut *ondatra.DUTDevice) bool {
+	deviationFlag := "deviation_traceroute_l4_protocol_udp"
+	if isFlagSetByUser(deviationFlag) {
+		log.Errorf("Value for %v is set using metadata.textproto. Flag value will be ignored!!!", deviationFlag)
+	}
+	return lookupDUTDeviations(dut).GetTracerouteL4ProtocolUdp()
```

Comment:
what do you think about passing in the name of the flag to "lookupDUTDeviations" so that the check doesn't need to be repeated in each accessor?

Reply:
lookupDUTDeviations will also be called by new deviations which would be added before we complete this migration. So, I thought we could keep this function seperate. 

I did move these lines to log the error to a seperate function `logErrorForFlag`. This function would only be called for those deviations which have a flag value and are migrated to use the metadata.textproto value. 

Let me know if that sounds ok.

Reply:
Sounds good.